src/components/DropZone/DropZone.tsx

```diff
@@ -541,7 +557,12 @@ export class DropZone extends React.Component<CombinedProps, State> {
     }
 
     this.dragTargets = this.dragTargets.filter((el: Node) => {
-      return el !== event.target && this.dropNode && this.dropNode.contains(el);
+      return (
+        el !== event.target &&
+        'current' in this.dropNode &&
```

Comment:
We can omit this check since we do it below

Reply:
`[ts] Object is possibly 'null'.`

Is the check necessary since,`in` just checks existence while `this.dropNode.current` checks whether it is null before trying to see whether it contains `el` ?